x-pack/plugins/security_solution/public/app/home/global_header/index.test.tsx

```diff
@@ -67,6 +68,21 @@ describe('global header', () => {
 
   beforeEach(() => {
     useVariationMock.mockReset();
+    (useKibana as jest.Mock).mockImplementation(() => {
+      return {
+        services: {
+          data: {
+            dataViews: {},
+          },
+          http: { basePath: { prepend: jest.fn((url: string) => url) } },
+          cloudExperiments: {
+            getVariation: jest.fn(),
+          },
+          theme: { theme$: jest.fn() },
+          setHeaderActionMenu: mockSetHeaderActionMenu,
+        },
+      };
+    });
```

Comment:
Can we add the mock for `setHeaderActionMenu: jest.fn()` in the generic kibana services mock? this way it will always be set for all tests, and we won't need to do explicit `mockImplementation` either.
https://github.com/angorayc/kibana/blob/f8d9cfccd94898885049d7b8dcbf5a38adcd43d9/x-pack/plugins/security_solution/public/common/lib/kibana/kibana_react.mock.ts#L126